### DISCUSSION TITLE
Is there a current status tracker?

### DISCUSSION THREAD

**Discussion by indigoviolet** [PaperWM-community/PaperWM#24] [Q&A]
I'm trying to understand the current state for different Gnome versions, and what might break if I upgrade. Is https://github.com/orgs/PaperWM-community/projects/1 the source of truth for this?

**Comment by smichel17** [comment]
I don't think there is a tracker at the moment. Not sure who created the board there or whether it is up to date. It does seem like a reasonable place to use as a source of truth, though.
